introduction.md

```diff
@@ -1 +1,9 @@
 Hello Sensei, I'm wajacps. I follow the path of JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?